open-metadata-implementation/access-services/cognos/cognos-server/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project.  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>cognos</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>1.8-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Cognos OMAS Server-side</name>
+    <description>
+        Server-side support for the Cognos Open Metadata Access Service (OMAS).
+    </description>
+
+    <artifactId>cognos-server</artifactId>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>admin-services-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>admin-services-registration</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>open-connector-framework</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>inmemory-repository-connector</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>repository-services-apis</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+
+       <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>cognos-api</artifactId>
+       </dependency>
+       <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>cognos-api</artifactId>
+			<version>${project.version}</version>
```

Comment:
we keep all versions in the `dependencyManagement` section of the top level pom - so the idea is you template the dependency there, and then in each sub-module pom do not include version, just group/artifact

Reply:
done https://github.com/odpi/egeria/pull/3044/commits/2c7f52af9a67feb496eedd853b634a9b5f478b2f